regression/cbmc/Malloc23/test.desc

```diff
@@ -7,6 +7,6 @@ pointer outside dynamic object bounds in \*p: FAILURE
 pointer outside dynamic object bounds in \*p: FAILURE
 pointer outside dynamic object bounds in p2\[.*1\]: FAILURE
 pointer outside dynamic object bounds in p2\[.*0\]: FAILURE
-\*\* 4 of 36 failed 
+\*\* 4 of .* failed 
```

Comment:
Not a big deal (and obviously similar changes happened elsewhere): could this please be `[0-9]+` instead of `.*`?

Reply:
done